modules/develop/pages/produce-data/configure-producers.adoc

```diff
@@ -87,215 +113,193 @@ to the majority of the brokers responsible for the partition in the cluster. As
-In cases when you configure the producer to be xref:./idempotent-producers.adoc[idempotent],
+When you configure the producer to be xref:./idempotent-producers.adoc[idempotent],
 up to five requests can be guaranteed to be in flight with the order preserved.
 
-==== `enable.idempotence`
+=== `enable.idempotence`
 
 To enable idempotence, set `enable.idempotence` to `true` (the default) in your
 Redpanda configuration.
 
-When idempotence is enabled, the producer will try to make sure that exactly one
+When idempotence is enabled, the producer tries to ensure that exactly one
 copy of every message is written to the broker. When set to `false`, the producer
 retries sending a message for any reason (such as transient errors like brokers
-not being available or not enough replicas exception), it can lead to duplicates.
+not being available or not enough replicas exception), and it can lead to duplicates.
 
 In most client libraries `enable.idempotence` is set to true by default.
 Internally, this is implemented using a special identifier that is assigned to
-every producer (the producer ID or "`PID`"). This ID, along with a "sequence
-number", is included in every message that is being sent to the broker. The
-broker will check if the PID/sequence number combination is larger than the
-previous one and, if not, it will discard the message.
+every producer (the producer ID or PID). This ID, along with a sequence
+number, is included in every message sent to the broker. The
+broker checks if the PID/sequence number combination is larger than the
+previous one and, if not, it discards the message.
 
-To guarantee true idempotent behavior, you must also set `acks=all` to ensure
+To guarantee true idempotent behavior, you must also set `acks=all` to ensure that
 all brokers record messages in order, even in the event of node failures.
 In this configuration, both the producer and the broker prefer safety and
 durability over throughput.
 
-Idempotence is only guaranteed within a session. A session starts once the
-producer is instantiated and a connection is established between the client and
+Idempotence is only guaranteed within a session. A session starts after the
+producer is instantiated and a connection is established between the client and the
 Redpanda broker. When the connection is closed, the session ends.
 
-If your application code retries a request, the producer client will assign a
+If your application code retries a request, the producer client assigns a
 new ID to that request, which may lead to duplicate messages.
 
-=== Message batching
+== Write caching
+
+Write caching lets Kafka clients produce to and consume data from topics without explicitly first flushing (fsync) the data to disk, even with `acks=all`. Clients are acknowledged as soon as writes are buffered on the desired set of replicas, following the `acks` level. Write caching provides lower latency and higher write throughput at the expense of lower durability guarantees. 
```

Comment:
rephrase a bit?

> Write caching allows Kafka clients to write data without explicitly fsyncing it to disk  when using (acks=all)

write caching only applies to acks=all, we need to clarify that. 

> desired set of replicas

replace with "majority of rpelicas"

remove "following the `acks` level", also remove references to higher throughput .. just say higher write performance at the expense of lower durability guarantees.